client/pages/editntc/editntc.js

```diff
@@ -1,6 +1,11 @@
 // pages/userinfo/userinfo.js
 Page({
 
+  formSubmit: function (e) {
+    console.log('the form of editing notice was submitted：', e.detail.value)
```

Comment:
I guess this is for testing purpose. It's not a great idea to expose your log messages in production code.